### PR TITLE
fixes #11128 - prevents `const_missing` error, BZ1241343

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

File: utils/bin/katello-disconnected [hunk 1]
```diff
@@ -16,6 +16,12 @@
 # workaround for: https://github.com/grosser/fast_gettext/issues/32
 Encoding.default_external='UTF-8' if RUBY_VERSION >= '1.9'
 
+# mad h4x0rz for BZ 1241343
+# solution reference: http://lists.openshift.redhat.com/openshift-archives/dev/2014-July/msg00193.html
+ENV['LD_LIBRARY_PATH'] += ':/opt/rh/v8314/root/usr/lib64'
+require 'action_view'
+require 'v8'
+
```
Comment (mccun934) [inline utils/bin/katello-disconnected]: minor beef, any reason these requires aren't next to the rest of the listand sit above the APP_NAME declaration?

Comment (komidore64) [inline utils/bin/katello-disconnected]: other than they're required solely to fix this bug, no.

Comment (komidore64) [inline utils/bin/katello-disconnected]: @mccun934 addressed

**Comment by mccun934** [comment]
ACK

**Pull request merged**